Client/src/Components/Graph/GraphView.tsx

```diff
@@ -205,9 +321,94 @@ export default function GraphView() {
                 <DatasetsList datasets={completeDatasets} onRemoveDatasetClick={onRemoveDataset} />
               </Grid>
             </Box>
+            <Grid container direction='row'>
+              <Grid item xs={12}>
+                <Button id='settingsToggle' variant="contained" onClick={handleSettingsClick} color="primary">Settings
+              {showSettings ? <KeyboardArrowUpIcon /> : <KeyboardArrowDownIcon />}
+                </Button>
+              </Grid>
+              {showSettings &&
+                <>
+                  <Grid item xs={4}>
+                    <FormControl>
+                      <InputLabel id="xVariable">X Variable</InputLabel>
+                      <Select
+                        labelId="xVariable"
+                        id="xVariable"
+                        value={xVariableName}
+                        autoWidth={true}
+                        onChange={handleXVariableChange}
+                      >
+                        {variables.map(type => (
+                          <MenuItem value={type}>{type}</MenuItem>

```

Comment:
can all this code be in its own component, we need to clean up current graphView its very heavy

Reply:
Small thing, add border? Looks a bit strange to just have it pop out of nowhere. 
![image](https://user-images.githubusercontent.com/25015601/105648902-de63c180-5e7b-11eb-97d0-ccbc25985f83.png)